### PR TITLE
Changelog cleanup

### PR DESCRIPTION
**Pull request by ernopp**
Cleanup ahead of 0.9.0-preview

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,38 +14,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - The `GenerateSchema`, `GenerateSchemaAndSnapshots`, and `CookAndGenerateSchema` commandlets can be invoked with the `-AdditionalSchemaCompilerArguments="..."` command line switch to output additional compiled schema formats. If no such switch is provided, only the schema descriptor will be produced. This switch's value should be a subset of the arguments that can be passed to the schema compiler directly (e.g., `--bundle_out="path/to/bundle.sb"`). A full list of possibles values is available via the [schema compiler documentation](https://docs.improbable.io/reference/14.2/shared/schema/introduction#schema-compiler-cli-reference)
 - Added the AllowUnresolvedParameters function flag that disables warnings for processing RPCs with unresolved parameters. This flag can be enabled through Blueprints or by adding a tag to the `UFUNCTION` macro.
 - Improved logging around entity creation.
-- Unreal Engine `4.23.1` is now supported. You can find the `4.23.1` version of our engine fork [here](https://github.com/improbableio/UnrealEngine/tree/4.23-SpatialOSUnrealGDK).
```
Comment (m-samiec) [inline CHANGELOG.md]: Why remove this? Because we're going to be supporting 4.24?

Comment (ernopp) [reply CHANGELOG.md]: no that was my mistake

File: CHANGELOG.md [hunk 2]
```diff
@@ -94,9 +83,24 @@ Usage: `DeploymentLauncher createsim <project-name> <assembly-name> <target-depl
 - Fixed an issue where newly created subobjects would have empty state when RepNotify was called for a property pointing to that subobject.
 - Fixed an issue where deleted, initially dormant startup actors would still be present on other workers.
 - Force activation of RPC ring buffer when load balancing is enabled, to allow RPC handover when authority changes
+- Clients will now validate schema against the server and log a warning if they do not match.
+- Starting a local deployment now checks if the required runtime port is blocked and allows the user to kill it
```
Comment (oblm) [inline CHANGELOG.md]:
```suggestion
- Starting a local deployment now checks if the required runtime port is blocked and allows the user to kill it.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -55,9 +44,9 @@ Usage: `DeploymentLauncher createsim <project-name> <assembly-name> <target-depl
...
 - The `InstallGDK` scripts now `git clone` the correct version of the `UnrealGDK` and `UnrealGDKExampleProject` for the `UnrealEngine` branch you have checked out. They read `UnrealGDKVersion.txt` & `UnrealGDKExampleProjectVersion.txt` to determine what the correct branches are.
-- Unreal Engine `4.24.3` is now supported. You can find the `4.24.3` version of our engine fork [here](https://github.com/improbableio/UnrealEngine/tree/4.24-SpatialOSUnrealGDK).
-- Enabling the Unreal GDK load balancer now creates a single query per server worker, depending on the defined load balancing strategy.
 - The `bEnableServerQBI` property has been removed, and the flag `--OverrideServerInterest` has been removed.
+- Unreal Engine `4.23.1` is now supported. You can find the `4.23.1` version of our engine fork [here](https://github.com/improbableio/UnrealEngine/tree/4.23-SpatialOSUnrealGDK). Please be aware of our [UE version support policy](https://documentation.improbable.io/gdk-for-unreal/docs/versioning-scheme#section-unreal-engine-version-support).
```
Comment (oblm) [inline CHANGELOG.md]:
```suggestion
- Unreal Engine `4.23.1` is now supported. You can find the `4.23.1` version of our engine fork [here](https://github.com/improbableio/UnrealEngine/tree/4.23-SpatialOSUnrealGDK-preview). Please be aware of our [UE version support policy](https://documentation.improbable.io/gdk-for-unreal/docs/versioning-scheme#section-unreal-engine-version-support). 
```

Comment (oblm) [reply CHANGELOG.md]: This points to our development branch, it should point to our stable preview branch, which currently does not exist but will when we ship `0.9.0-preview`.

File: CHANGELOG.md [hunk 4]
```diff
@@ -55,9 +44,9 @@ Usage: `DeploymentLauncher createsim <project-name> <assembly-name> <target-depl
...
-- Unreal Engine `4.24.3` is now supported. You can find the `4.24.3` version of our engine fork [here](https://github.com/improbableio/UnrealEngine/tree/4.24-SpatialOSUnrealGDK).
-- Enabling the Unreal GDK load balancer now creates a single query per server worker, depending on the defined load balancing strategy.
 - The `bEnableServerQBI` property has been removed, and the flag `--OverrideServerInterest` has been removed.
+- Unreal Engine `4.23.1` is now supported. You can find the `4.23.1` version of our engine fork [here](https://github.com/improbableio/UnrealEngine/tree/4.23-SpatialOSUnrealGDK). Please be aware of our [UE version support policy](https://documentation.improbable.io/gdk-for-unreal/docs/versioning-scheme#section-unreal-engine-version-support). 
+- Unreal Engine `4.24.3` is now supported. You can find the `4.24.3` version of our engine fork [here](https://github.com/improbableio/UnrealEngine/tree/4.24-SpatialOSUnrealGDK). Please be aware of our [UE version support policy](https://documentation.improbable.io/gdk-for-unreal/docs/versioning-scheme#section-unreal-engine-version-support).
```
Comment (oblm) [inline CHANGELOG.md]:
```suggestion
- Unreal Engine `4.24.3` is now supported. You can find the `4.24.3` version of our engine fork [here](https://github.com/improbableio/UnrealEngine/tree/4.24-SpatialOSUnrealGDK-preview). Please be aware of our [UE version support policy](https://documentation.improbable.io/gdk-for-unreal/docs/versioning-scheme#section-unreal-engine-version-support). 
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -55,9 +44,9 @@ Usage: `DeploymentLauncher createsim <project-name> <assembly-name> <target-depl
...
 - The `InstallGDK` scripts now `git clone` the correct version of the `UnrealGDK` and `UnrealGDKExampleProject` for the `UnrealEngine` branch you have checked out. They read `UnrealGDKVersion.txt` & `UnrealGDKExampleProjectVersion.txt` to determine what the correct branches are.
-- Unreal Engine `4.24.3` is now supported. You can find the `4.24.3` version of our engine fork [here](https://github.com/improbableio/UnrealEngine/tree/4.24-SpatialOSUnrealGDK).
-- Enabling the Unreal GDK load balancer now creates a single query per server worker, depending on the defined load balancing strategy.
 - The `bEnableServerQBI` property has been removed, and the flag `--OverrideServerInterest` has been removed.
+- Unreal Engine `4.23.1` is now supported. You can find the `4.23.1` version of our engine fork [here](https://github.com/improbableio/UnrealEngine/tree/4.23-SpatialOSUnrealGDK-preview). Please be aware of our [UE version support policy](https://documentation.improbable.io/gdk-for-unreal/docs/versioning-scheme#section-unreal-engine-version-support).
```
Comment (m-samiec) [inline CHANGELOG.md]:
```suggestion
- Unreal Engine `4.23.1` is now supported. You can find the `4.23.1` version of our engine fork [here](https://github.com/improbableio/UnrealEngine/tree/4.23-SpatialOSUnrealGDK-release). Please be aware of our [UE version support policy](https://documentation.improbable.io/gdk-for-unreal/docs/versioning-scheme#section-unreal-engine-version-support). 
```

Comment (m-samiec) [reply CHANGELOG.md]: ?

**Comment by joshuahuburn** [comment]
Closing this as handled in the 0.9.0-rc

**Pull request closed without merging**